### PR TITLE
COMMERCE-574 Fixed shipments admin portlet

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test

**Comment by ethanbustad** [comment]
@Commando18 can you run `gradlew baseline`, and commit and push the changes? That should fix the CI failure.

**Comment by alessiorendina** [comment]
ci:retest

**Comment by alessiorendina** [comment]
@ethanbustad did it, thanks!

**Comment by ethanbustad** [comment]
@Commando18 looks like it needs `gradlew formatSource`, too :/

There's a chance the above failure doesn't show up with `formatSource` locally. If so, trust the CI version for now and make the change manually -- I'm trying to get it resolved so we get the same results in CI and locally.

**Comment by alessiorendina** [comment]
ci:retest

File: commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentLocalServiceImpl.java [hunk 1]
```diff
@@ -175,6 +176,58 @@ public CommerceShipment updateCommerceShipment(
 			int expectedDateHour, int expectedDateMinute)
 		throws PortalException {
 
+		CommerceShipment commerceShipment =
```
Comment (ethanbustad) [inline commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentLocalServiceImpl.java]: Can you sort this down below `String phoneNumber = null;`?

File: commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentLocalServiceImpl.java [hunk 2]
```diff
@@ -274,6 +334,41 @@ else if (status ==
...
+			String street1, String street2, String street3, String city,
+			String zip, long commerceRegionId, long commerceCountryId,
+			String phoneNumber)
+		throws PortalException {
+
+		CommerceAddress commerceAddress =
+			commerceShipment.fetchCommerceAddress();
+
+		if ((commerceAddress != null) &&
+			name.equals(commerceAddress.getName()) &&
```
Comment (ethanbustad) [inline commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentLocalServiceImpl.java]: Couldn't these variables be passed in as `null`? We check for nullness in `commerceAddressLocalService.addCommerceAddress`, so it seems like we shouldn't break here if these values are `null`. We could just sidestep the issue with `Objects.equals`.

File: commerce-shipment-web/src/main/resources/META-INF/resources/shipment/details.jsp [hunk 3]
```diff
@@ -21,6 +21,16 @@ CommerceShipmentItemDisplayContext commerceShipmentItemDisplayContext = (Commerc
 
 CommerceShipment commerceShipment = commerceShipmentItemDisplayContext.getCommerceShipment();
 long commerceShipmentId = commerceShipmentItemDisplayContext.getCommerceShipmentId();
+
+CommerceAddress commerceAddress = commerceShipment.fetchCommerceAddress();
```
Comment (ethanbustad) [inline commerce-shipment-web/src/main/resources/META-INF/resources/shipment/details.jsp]: Can you sort this below `long commerceRegionId = 0;`?

File: commerce-shipment-web/src/main/resources/META-INF/resources/shipment/details.jsp [hunk 4]
```diff
@@ -65,4 +136,45 @@ long commerceShipmentId = commerceShipmentItemDisplayContext.getCommerceShipment
...
+						},
+						callback
+					);
+				},
+				selectDesc: 'nameCurrentValue',
+				selectId: 'commerceCountryId',
+				selectSort: '<%= true %>',
+				selectVal: '<%= commerceCountryId %>'
+			},
+				{
```
Comment (ethanbustad) [inline commerce-shipment-web/src/main/resources/META-INF/resources/shipment/details.jsp]: Indentation is a bit off here (162-172)

File: commerce-shipping-web/src/main/resources/META-INF/resources/edit_shipping_settings.jsp [hunk 5]
```diff
@@ -18,27 +18,66 @@
...
-<portlet:actionURL name="editCommerceShippingSettings" var="editCommerceShippingSettingsActionURL" />
+<div class="container-fluid-1280">
+	<portlet:actionURL name="editCommerceShippingSettings" var="editCommerceShippingSettingsActionURL" />
+
+	<%
+	for (Map.Entry<String, CommerceShippingOriginLocator> entry : commerceShippingOriginLocators.entrySet()) {
+		String key = entry.getKey();
+		CommerceShippingOriginLocator commerceShippingOriginLocator = entry.getValue();
+
+		boolean checked = false;
```
Comment (ethanbustad) [inline commerce-shipping-web/src/main/resources/META-INF/resources/edit_shipping_settings.jsp]: Ideally this should be sorted above `String key = entry.getKey();`.

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by alessiorendina** [comment]
ci:retest

**Comment by alessiorendina** [comment]
ci:retest

**Pull request merged**